code/modules/client/preferences.dm

```diff
@@ -1159,7 +1164,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 						b_type = new_b_type
 
 				if("hair")
-					if(species == "Human" || species == "Unathi" || species == "Tajaran" || species == "Skrell" || species == "Machine" || species == "Vulpkanin")
+					if(species in list("Human", "Unathi", "Tajaran", "Skrell", "Machine", "Vulpkanin", "Vox"))

```

Comment:
Many Vox already colour their hair at mirrors and with dye and such, may as well let them do it at character creation.